### ISSUE TITLE
🔍 Accessibility Issue: region - MODERATE Impact

### ISSUE THREAD

**Issue by FarukDelic** [FarukDelic/Axe-Python-Demo#6]
## Accessibility Violation Found
    
**Rule ID:** `region`  
**Impact:** MODERATE  
**URL Tested:** https://www.google.com  
**Description:** Ensures all page content is contained by landmarks

### Help Information
All page content should be contained by landmarks

**Help URL:** https://dequeuniversity.com/rules/axe/4.4/region?application=axeAPI

#### Element 1
- **HTML:** `<div id="SIvCob">`
- **Target:** `#SIvCob`
- **Failure Summary:** Fix any of the following:
  Some page content is not contained by landmarks
- **Any of these checks failed:**
  - Some page content is not contained by landmarks

### How to Fix
Please refer to the [Axe Rules documentation](https://dequeuniversity.com/rules/axe/4.4/region?application=axeAPI) for detailed guidance on resolving this accessibility issue.

---
*This issue was automatically created by the Axe Accessibility Test workflow.*
